docs/xaml/live-visual-tree.md

```diff
@@ -0,0 +1,49 @@
+ms.date: 06/23/2022
+---
+
+# Live Visual Tree for .NET MAUI
+
+.NET Multi-platform App UI (.NET MAUI) Live Visual Tree is a Visual Studio feature that provides a tree view of the UI elements in your running .NET MAUI app.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+When your .NET MAUI app is running in debug configuration, with the debugger attached, the Live Visual Tree window can be opened by selecting **Debug > Windows > Live Visual Tree** from the Visual Studio menu bar:
+
+:::image type="content" source="media/live-visual-tree/live-visual-tree.png" alt-text="Screenshot of the Live Visual Tree window.":::
+
+Provided that Hot Reload is enabled, the **Live Visual Tree** window will display the hierarchy of your app's UI elements regardless of whether the app's UI is built using XAML or C#. However, you will have to disable Just My XAML to display the hierarchy of your app's UI elements for UIs built using C#.
+
+## Just My XAML
+
+The view of the UI elements is simplified by default using a feature called Just My XAML. Clicking the **Show Just My XAML** button shows all UI elements in the visual tree:
+
+:::image type="content" source="media/live-visual-tree/just-my-xaml.png" alt-text="Screenshot of all UI elements in the Live Visual Tree window.":::
+
+Just My XAML can be disabled by selecting **Debug > Options > XAML Hot Reload** from the Visual Studio menu bar. Next, in the **Options** dialog box, ensure that **Enable Just My XAML in Live Visual Tree** is disabled:
+
+:::image type="content" source="media/live-visual-tree/just-my-xaml-disabled.png" alt-text="Screenshot of Just My XAML button disabled in the Live Visual Tree window.":::
+
+## Find a UI element
+
+The structure of a XAML UI has a lot of elements that you're probably not directly interested in, and if you don't have a full understanding of the app's source code you might have a difficult time navigating the visual tree to find what you're looking for. Therefore, the **Live Visual Tree** window has multiple approaches that let you use the app's UI to help you find the element you want to examine:
+
+- **Select element in the running application**. You can enable this mode by clicking the **Select Element in the Running Application** button in the **Live Visual Tree** toolbar:
+
+  :::image type="content" source="media/live-visual-tree/select-element-in-running-app.png" alt-text="Screenshot of the Select Element in the Running Application button in the Live Visual Tree toolbar.":::
+
+  With this mode enabled, when you can select a UI element in the app the **Live Visual Tree** window automatically updates to show the node in the tree corresponding to that element.
+
+- **Display layout adorners in the running application**. You can enable this mode by clicking the **Display Layout Adorners in the Running Application** button in the **Live Visual Tree** toolbar:
```

Comment:
This doesn't seem to do anything.

Reply:
There is a bug with the current version of MAUI for WinUI where Adorners may not be drawn (https://github.com/dotnet/maui/issues/8096). It should work for Android and iOS though. If it doesn't, it needs to be filed as a bug, since it should work.